### ISSUE TITLE
One Samsung SSD works, the other doesn't.

### ISSUE THREAD

**Issue by SlagOMatic** [FIX94/Nintendont#994]
Got a 512GB Samsung 860 Pro in a generic made-in-China USB enclosure which works fine. Nintendont sees the drive and the Game Cube ISOs I've put on it and everything.

Got a 1TB Samsung T7 external drive which doesn't work fine. Nintendont doesn't even see the drive attached. (To clarify: Wiixplorer sees the hard drive and its contents, but Nintendont don't.)

Both drives are formatted as FAT (not ExFAT) and have the same folder hierarchy. I've tried both USB ports, connecting before/after the Wii is powered up, connecting before/after Nintendont is launched, all the same. I don't see the drive on the [compatibility list](https://wiki.gbatemp.net/wiki/USB_Devices_Compatibility_List) so I might be in new ground here. Any ideas?

**Comment by carnage702** [comment]
are they both mbr partition tables?

**Comment by SlagOMatic** [comment]
Yes, both MBR.

>

**Comment by carnage702** [comment]
> Yes, both MBR.
> […](#)
> On Mar 24, 2022, at 2:47 AM, pedro702 ***@***.***> wrote: ﻿ are they both mbr partition tables? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.

Maybe it's just incompatible, I heard some recent drives have issues with Homebrew in general.

**Comment by SlagOMatic** [comment]
Is there anything I can do/contribute to helping make this work?

**Comment by Special-Niewbie** [comment]
> Is there anything I can do/contribute to helping make this work?

Because it's not an Y cable from the 1TB Samsung T7.......

**Comment by SlagOMatic** [comment]
>Because it's not an Y cable from the 1TB Samsung T7.......

Sorry, explain? I mean, I know what a Y-cable is, but how would the lack of a Y-cable in this instance mean that Wiixplorer can see and navigate the drive without issue (which means that the Wii itself sees the drive) but Nintendont cannot?

**Comment by carnage702** [comment]
> > Because it's not an Y cable from the 1TB Samsung T7.......
> 
> Sorry, explain? I mean, I know what a Y-cable is, but how would the lack of a Y-cable in this instance mean that Wiixplorer can see and navigate the drive without issue (which means that the Wii itself sees the drive) but Nintendont cannot?

sometimes nintendont needs more power due to drawing more fast calls than even wii games running trough usbloadergx so a y cable is always something one could try tough.

**Comment by SlagOMatic** [comment]
I'm sorry, I'm sure I must be mistaken. Are you saying that because Nintendont allegedly "needs more power due to drawing more fast calls" that it actually affects the output power of the USB ports? That's like saying "You can't run Adobe Premiere with a USB hard drive because Premiere takes up too much power." Where is your citation to support this claim?

I've been playing and working with computers for nearly 45 years and I've NEVER heard of any such thing.

**Comment by carnage702** [comment]
> I'm sorry, I'm sure I must be mistaken. Are you saying that because Nintendont allegedly "needs more power due to drawing more fast calls" that it actually affects the output power of the USB ports? That's like saying "You can't run Adobe Premiere with a USB hard drive because Premiere takes up too much power." Where is your citation to support this claim?
> 
> I've been playing and working with computers for nearly 45 years and I've NEVER heard of any such thing.

well on wiiu the usb power is not sufficient for many hdds that why they need the y cable, while a single cable hdd works on wiiu for wii games and even wiiu games they are on the edge of failling nintendo itself recomends external powered hdds or y cables on their faqs due to wii and wiiu power from usb being very poor


Does a Wii U need Y cable?

Nintendo recommends using self-powered hard disk drives, i.e. devices that use an external power supply instead of drawing power through USB. So we cannot guarantee that such devices will run properly on the Wii U without the use of a supplemental power cable called a y-cable that draws power from two USB ports.

I assume the same can happen on wii depending on the power draw of each hdd, nintendont for sure never works on wiiu without self powered hdds or y cables it has been confirmed by alot of people.

**Comment by SlagOMatic** [comment]
>well on wiiu the usb power is not sufficient for many hdds that why they need the y cable, while a single cable hdd works on wiiu for wii games and even wiiu games they are on the edge of failling

This part makes perfect sense and is in line with the entirety of the computing world. If a device does not have sufficient power from USB, the device won't work and therefore the host system (the computer) wouldn't see the device. And if the host system doesn't see the device then no software running on the device would be able to see or access it.

But that's not what's happening here.

If the device (the SSD, in this case) was not getting sufficient power from USB then the host system (the Wii, in this case) would not be able to see the device. But the Wii _does_ see the device which means that the device is getting sufficient power from USB.

Your previous comment asserted that Nintendont "needs more power" and that's why it doesn't see the drive. This is absolutely, positively, 100% _NOT_ how computers (or game consoles) work.

**Comment by carnage702** [comment]
> > well on wiiu the usb power is not sufficient for many hdds that why they need the y cable, while a single cable hdd works on wiiu for wii games and even wiiu games they are on the edge of failling
> 
> This part makes perfect sense and is in line with the entirety of the computing world. If a device does not have sufficient power from USB, the device won't work and therefore the host system (the computer) wouldn't see the device. And if the host system doesn't see the device then no software running on the device would be able to see or access it.
> 
> But that's not what's happening here.
> 
> If the device (the SSD, in this case) was not getting sufficient power from USB then the host system (the Wii, in this case) would not be able to see the device. But the Wii _does_ see the device which means that the device is getting sufficient power from USB. It's Nintendont, _not_ the Wii, that doesn't see the device.
> 
> Your previous comment asserted that Nintendont "needs more power" and that's why it doesn't see the drive. This is absolutely, positively, 100% _NOT_ how computers (or game consoles) work.

its all the part of the stability,  nintendont is more unstable on the calls than regular wii games or wiiu games thats why many hdds need its own power source or an y cable because they are already unstable when running through the crappy wii/wiiu usb ports that generate poor voltage, just get an y cable and try it, if you want, if you dont then no problem for me, just do whatever you want, we gave you a possible solution, its up to you to do what you want.

**Comment by SlagOMatic** [comment]
>its all the part of the stability, nintendont is more unstable on the calls than regular wii games or wiiu games...

The stability of a program does not, never has, and almost certainly never will, affect the power flow of a USB port. I _still_ don't know what you're basing this truly strange argument from. Again I ask you: What are your sources for the information you are giving here?

For shits and giggles I tried the Y-cable and, as expected and by every rational and logical rule of computing of this type, it did not solve the issue. Until and unless you can cite legitimate sources for your claims, what you're suggesting is laughable at best and outright disinformation at worst.

**Comment by carnage702** [comment]
> > its all the part of the stability, nintendont is more unstable on the calls than regular wii games or wiiu games...
> 
> The stability of a program does not, never has, and almost certainly never will, affect the power flow of a USB port. I _still_ don't know on what you're basing this truly strange argument. Again I ask you: What are your sources for the information you are giving here?
> 
> For shits and giggles I tried the Y-cable and, as expected and by every rational and logical rule of computing of this type, it did not solve the issue. Until and unless you can cite legitimate sources for your claims, what you're suggesting is laughable at best and outright disinformation at worst.

i just gave you a reputable source... nintendo itself advised on using a y cable or self power hdds  did you even read the link? this was posted by nintendo not me or a random internet person.

https://www.nintendo.co.uk/Support/Wii-U/Storage/Which-kind-of-USB-HDD-is-suitable-for-use-with-Wii-U-/Which-kind-of-USB-HDD-is-suitable-for-use-with-Wii-U-678127.html

and there are alot of people that couldnt get the hdd working with nintendont without the y cable even tough it worked with wii games normally. i have been managing nintendont for years so i saw hundreds off issues where y cables solved many people problems.

This is the exact same reason why wiiu gc adapter for smash bros has a y cable because the usb ports couldnt even power a controller and rumble at the same time even tough a regular gc port could do.

**Comment by SlagOMatic** [comment]
First: The link you provided has nothing to do with the Wii. It is about the Wii U, which (spoiler alert) is a completely different platform and not what we're talking about here.

Second: Yes, I read the article -- and I suggest you read it too since what you're saying has absolutely nothing to do with the article.

YOUR claim is that Nintendont requires so much electricity to run that it's not leaving enough electricity for the USB ports to power the SSD, while conveniently ignoring the irrefutable fact that the SSD has plenty of power and is seen by the Wii and Homebrew-based file managers across the board. Somehow, in defiance of literally every aspect of technology known to man and in a way which you have yet to explain or justify or cite sources for, you believe that a single app -- in this case Nintendont -- is so electronically demanding that it's _actually sucking power from the USB ports_ and that's the reason why Nintendont and Nintendont _alone_ doesn't see the SSD.

THE ARTICLE, on the other hand, states a well-known aspect of USB power that's been around for as long as USB has been around: a device (hard drive, in this case) which requires more power than a single USB port can provide may not be recognized by its host, and thereby may require a Y-cable and second USB port in order to supply that additional power.

You have inexplicably made the leap from "hardware needs more power than a single USB port can provide in order to be seen by the system" to "the software is sucking up all the electricity".  Your position is so profound in its ignorance that it would be laughable if it weren't for the fact that you seriously believe it.

So, for the final time: Can you cite any ACTUAL, RELEVANT resources to back up your claim?

**Comment by SlagOMatic** [comment]
Okay, so now that the signal:noise ratio has cleared up, I have additional information for anyone pursuing this adventure. (Spoiler: It has NOTHING to do with "software sucking up all the electricity" as some recent dimwittery has suggested.)

Over the past few weeks I've been experimenting with various drives and interfaces. Short summary:

Oh, first, clarification: In this context "doesn't work" means "the Wii sees the drive through Wiixplorer, but Nintendont doesn't see the drive at all".

* External SSDs that don't use a SATA interface (e.g., the Samsung T7, which is basically an M.2 drive and a bridge board inside an enclosure) have a substantially lower chance of working with Nintendont, which is to say I've tried several different brands and models and was unable to get any of them to work with Nintendont (even though the Wii sees all of them).
* External SSD that are made from a SATA-based SSD with a USB interface have a much better chance of working. However, not all enclosures work, and not all SATA-based SSDs work. I tried several enclosures from several manufacturers and the only ones that didn't work with Nintendont were enclosures that originally contained a SATA hard drive. (E.g., I have an enclosure that was originally a Seagate 2.5" USB hard drive, but the drive went south and I recovered the enclosure. While the enclosure works with every device I've tried it with, the combination of a SATA-based SSD + this enclosure is something that Nintendont doesn't see. Ditto for two other enclosures that originally held Western Digital hard drives.) Every USB 3 enclosure I tried, and all but one USB 2 enclosures I tried, work with Nintendont. The one USB 2 enclosure that didn't work was the oldest one I have and is, oddly enough, the only enclosure that used a USB Mini connection on the back of the enclosure.

More specifically...

* Samsung PRO and EVO drives (256, 512, 1TB) worked; Samsung QVO drives (256, 512) did NOT work.
* Western Digital Blue drives (512, 1TB) worked.
* SanDisk SSD Plus drives (512, 1TB) did NOT work.
* PNY SSD (1TB) did NOT work.
* Kingston SSDNow (64) did NOT work.
* Crucial BX100 SSD (512) worked.

Hope this helps.
